src/index.html

```diff
@@ -11,6 +11,55 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form
+      action="#"
+      method="post"
+      class="form"
+      data-qa="big"
+    >
+      <label for="input1">
+        <!-- <img
+          src="./images/Search.svg"
+          alt="search"
+          class="searchImg"
+        > -->
+        <button type="button" class="searchImg">
+        </button>
+
+        <input
+        type="text"
+        class="input bigInput"
+        id="input1"
+        placeholder="Try &#34Los Angeles&#34"
+        data-qa="keypress"
```

Comment:
add indents for attributes and run through [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md) once again